strawberry/schema/schema_converter.py

```diff
@@ -163,6 +175,38 @@ def from_input_field(self, field: StrawberryField) -> GraphQLInputField:
             deprecation_reason=field.deprecation_reason,
         )
 
+    FieldType = TypeVar("FieldType", GraphQLField, GraphQLInputField)
+
+    @staticmethod
+    def _get_thunk_mapping(
```

Comment:
Can you add a (brief) docstring documenting what this function does?

Reply:
Added one now 😄